rate-limit-cloud-controller-api.html.md.erb

```diff
@@ -35,3 +39,9 @@ The estimate is based on the fraction remaining on the Cloud Controller instance
 This might result in inconsistent values for the `X-RateLimit-Remaining` header when running multiple instances of Cloud Controller API, such as some requests still being allowed when the header value is `0`.
 
 When requests exceed the maximum rate limit value, the Cloud Controller API returns a `429: Too Many Requests` error code.
+
+## <a id="Rate Limit Responses: Service Brokers"></a> Rate Limit Responses: Service Brokers
+
```

Comment:
Should we write a sentence here that makes clear its limiting concurrency on broker requests so a statement when the counter for this is incremented and decremented  ? Just for people to better understand that this endpoint has exceptional rate limiting behaviour over the usual one that applies to all others but also including Service broker endpoints in addition.

Reply:
The rate limit is implemented on service related CF API endpoints, not exactly on service brokers. 
+1 for documenting the endpoints (see https://github.com/cloudfoundry/cloud_controller_ng/blob/main/middleware/service_broker_rate_limiter.rb#L61)